src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="box">
+        <div class="image">
+          <img src="./images/logo.png" alt="logo">
+        </div>
+        <nav class="link">
+          <ul class="link">
```

Comment:
```suggestion
        <nav class="nav">
          <ul class="nav__list">
```
use propper class naming